packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/aspnet/integrationsteps.md

```diff
@@ -110,26 +100,22 @@ switch (authnResponse.AuthenticationStatus)
 ...
 ```
 
-If the `VerifyAuthenticatorAsync` call is successful, it should return
-`AwaitingPasswordReset`. This status indicates that the user can now change their
-password. At this point, the user should be redirected to the change password page.
+If the `VerifyAuthenticatorAsync()` call is successful, then the `AwaitingPasswordReset` status is returned. This status indicates that the user can now change their password. At this point, redirect the user to the change password page.
 
 ### 8: Create the change password page
 
-Create a change password page that allows the user to enter the
-new password and initiate the change password.
+Create a change password page that allows the user to enter their new password and initiate the password change.
 
 <div class="common-image-format">
 
-![Displays an example change password page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-social-sign-in-link.png)
+![Displays an example change password page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-change-pwd.png)
 
 </div>
 
 ### 9: Make a call to the ChangePasswordAsync method
 
-The final step is to make a call to `ChangePasswordAsync` to change the
-user's password. Pass `ChangePasswordOptions` into the method call with
-the `NewPassword` property set to the new password.
+The final step is to make a call to the `ChangePasswordAsync()` method to change the
```

Comment:
weird that the sentence breaks right here....not a big deal...

Reply:
I'll combine the three separate lines for the sentence into one.